### PR TITLE
update README

### PR DESCRIPTION
**Pull request by omikron-fp**
The README for the experimental feature Predictive Basket.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,118 @@
 FACT-Finder Web Components
 ==========================
+Predictive Basket branch (experimental)
+---------------------------------------
+
+This branch introduces the `ff-predictive-basket` element for easy integration with the new FACT-Finder Predictive Basket feature. To use it, you need FACT-Finder version **7.3** or higher.
```
Comment (toarm) [inline README.md]:
```suggestion
This branch introduces the `ff-predictive-basket` element for easy integration with the new FACT-Finder Predictive Basket feature. To use it, you need the latest FACT-Finder version **7.3** or higher.
```

File: README.md [hunk 2]
```diff
@@ -1,20 +1,118 @@
...
+### The Predictive Basket Element
+
+The following listing shows the newly introduced element. The element itself does not have a visual component and must be used in combination with an `ff-record-list` in the same manner as `ff-recommendation`. See further below for more details.
+
+```html
+<ff-predictive-basket user-id="123456"
+                      max-results="10"
+                      certainty="60"
+                      blacklist="art-id1,art-id2">
+    <!-- requires ff-record-list here to work -->
```
Comment (toarm) [inline README.md:19]:
```suggestion
    <ff-record-list subscribe="false">
        <ff-record>
            <img data-image="{{record.image}}" data-image-onerror="../img_not_found.gif">
            <div>
                <a data-anchor="{{record.deeplink}}" data-redirect="{{record.deeplink}}">
                    {{record.name}}
                </a>
                <div>Price: <strong>{{record.price}}</strong></div>
            </div>
        </ff-record>
    </ff-record-list>
```

File: README.md [hunk 3]
```diff
@@ -1,20 +1,118 @@
...
+
+Once the element is correctly initialized, it will automatically issue a request to FACT-Finder and display the result. This will happen only once per page load. If you need to refresh the displayed data, refer to the `getPredictions()` method below.
+
+#### Attributes
+
+Attribute | Required | Type | Default | JS property
+--------- | -------- | ---- | ------- | -----------
+user-id   | yes      | String | _(none)_ | `userId`
+
+The ID of the user for whom products shall be predicted. If the user ID is not specified, no request to FACT-Finder will be issued.
```
Comment (toarm) [inline README.md]: I would mention explicitly that supplying or changing the userId will issue an request.

File: README.md [hunk 4]
```diff
@@ -1,20 +1,118 @@
...
+
+
+
+Attribute | Required | Type | Default | JS property
+--------- | -------- | ---- | ------- | -----------
+max-results | optional | Number |  _(unlimited)_ | `maxResults`
+
+The maximum amount of products to be returned from FACT-Finder. To receive all predictions, omit this attribute or set its value to `-1`.
```
Comment (toarm) [inline README.md:39]: maybe 10 or 5 as default?

File: README.md [hunk 5]
```diff
@@ -1,20 +1,118 @@
...
+        </ff-record>
+    </ff-record-list>
+</ff-predictive-basket>
+```
+
+#### Manual update with `blacklist`
+```js
+function onUpdate(itemsToExclude) {
+    const predBasket = document.querySelector("ff-predictive-basket");
+    predBasket.blacklist = itemsToExclude;  // itemsToExclude must be a comma-separated list
```
Comment (toarm) [inline README.md]:
```suggestion
    predBasket.blacklist = ["blacklistedId1", "blacklistedId2", "blacklistedId3"];  // itemsToExclude must be a comma-separated list
```

Comment (toarm) [reply README.md]: my suggestion doesnt make sense in the context of this function x/ but think showing the the data type of itemsToExclude more verbose makes sense.

File: README.md (hunk 4 above)
Comment (omikron-fp) [reply README.md:39]: Default is according to FF API

File: README.md (hunk 2 above)
Comment (omikron-fp) [reply README.md:19]: Purpose is to focus on the new stuff only. Complete example is further below.

File: README.md (hunk 4 above)
Comment (toarm) [reply README.md:39]: then we should mention that or people probably end up confused

**Review by toarm — APPROVED** [review]

**Pull request merged**